demos/graphql/README.rst

```diff
@@ -0,0 +1,5 @@
+pip install -r requirements-dev.txt
```

Comment:
Probably good idea to update README early with information how to start app, as well as purpose of the app.